### ISSUE TITLE
Delivery delay of an image file with ?recache option >20sec!

### ISSUE THREAD

**Issue by Michaelsy** [dokuwiki/dokuwiki#3930]
Delivery delay of an image file with `?recache` option >20sec.
See [demo](https://www.dokuwiki.org/playground:recache-demo).

\- Michael Sy.

**Comment by splitbrain** [comment]
Yeah I have noticed this before. Unfortunately it only happens with this image hosted at cosmocode and only on the dokuwiki server. I cannot reproduce it locally which makes debugging hard. The problem seems to be related to downloading the image, not to caching.

**Comment by Michaelsy** [comment]
You are right, also no problem on my IONOS/1&1 webspace and DokuWiki on a Stick.

\. Michael Sy.

**Comment by alexgearbox** [comment]
Do we need to not use the "recache" option for this image (https://www.cosmocode.de/static/img/dokuwiki/dwplugins.png) on dokuwiki.org?
